tests/Directory.Build.targets

```diff
@@ -0,0 +1,11 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<ItemGroup Condition="'$(CUSTOM_DOTNET_VERSION)' != ''">
+		<FrameworkReference Update="Microsoft.NETCore.App" RuntimeFrameworkVersion="$(CUSTOM_DOTNET_VERSION)" />
+		<KnownFrameworkReference Update="Microsoft.NETCore.App" TargetingPackVersion="$(CUSTOM_DOTNET_VERSION)" />
+		<KnownFrameworkReference Update="Microsoft.NETCore.App" RuntimeFrameworkVersion="$(CUSTOM_DOTNET_VERSION)" />
+
+		<FrameworkReference Update="Microsoft.NETCore.App.Mono" RuntimeFrameworkVersion="$(CUSTOM_DOTNET_VERSION)" />
+		<KnownFrameworkReference Update="Microsoft.NETCore.App.Mono" TargetingPackVersion="$(CUSTOM_DOTNET_VERSION)" />
+		<KnownFrameworkReference Update="Microsoft.NETCore.App.Mono" RuntimeFrameworkVersion="$(CUSTOM_DOTNET_VERSION)" />
```

Comment:
@rolfbjarne curious, why did you need this? we don't produce a package with that name afaik.

Reply:
The idea was to pick up the runtime packs from the locally build dotnet/runtime, but it wouldn't surprise me if this isn't necessary (or doesn't do anything useful), it took me a while and some stumbling to figure out how to make the build pick up the locally built runtime packs, and this might have been a shot in the dark that missed completely.

Reply:
Ok, yeah this shouldn't do anything and can be removed.

Here's how we redirect the runtime pack location to a folder in dotnet/runtime: https://github.com/dotnet/runtime/blob/bdc5d79e9976aeeeabe2a6e38b1e80e33bd01506/eng/targetingpacks.targets#L116-L117